js/markDone.js

```diff
@@ -0,0 +1,25 @@
+import { EDIT, COMPLETE } from "/js/constants.js";
+
+export default function completeToDo(element, taskList, id) {
```

Comment:
```suggestion
export default function markToDoAsCompleted(element, taskList, id) {
```

Reply:
I have renamed it to markTodoAsCompleted as suggested.